tests/behavioural/features/steps/message_put.py

```diff
@@ -373,14 +375,31 @@ def step_impl(context):
 def step_impl(context):
     headers['user_urn'] = 'respondent.123'
     data['urn_from'] = headers['user_urn']
-    response = app.test_client().post("http://localhost:5050/draft/save", data=json.dumps(data), headers=headers)
+    response = app.test_client().post("http://localhost:5050/draft/save",
+                                      data=json.dumps(data), headers=headers)
     post_resp = json.loads(response.data)
     context.msg_id = post_resp['msg_id']
 
```

Comment:
This only checks the message id , not if any of the data has changed as expected ?